ArduCopter/config.h

```diff
@@ -147,7 +147,7 @@
 //////////////////////////////////////////////////////////////////////////////
 // winch support
 #ifndef WINCH_ENABLED
```

Comment:
maybe we can also replace WINCH_ENABLED with AP_WINCH_ENABLED?

Reply:
Hmmm.  For some reason I wasn't game to do that.

I've pushed up another commit which does it - and the results from build it are the same as above (i.e. no compile-time changes apart from minimized boards which get another line in their hwdef

Reply:
(oh and SkyViper gets a little more in its hwdef too)